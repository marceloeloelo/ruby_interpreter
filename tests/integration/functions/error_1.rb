def method_1()
end

method_111111()
