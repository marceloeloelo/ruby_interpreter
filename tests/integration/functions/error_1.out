NameError: undefined local variable or method `method_111111' for main:Object
